### PR TITLE
Added Various mods, Bash Tags, DLC, and TTW

### PR DESCRIPTION
**Pull request by darthbdaman**
Added JSawyer and JSawyer - Compatibility Edition

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -714,6 +714,45 @@ plugins:
       - NPC.Class
       - NPC.Race
       - Scripts
+  - name: 'jsawyer.esp'
+    url: [ 'http://diogenes-lamp.info/jsawyer_fnv_mod.zip' ]
+    priority: -400800000
```
Comment (Freso) [inline masterlist.yaml]: Any reason for this `priority`?

File: masterlist.yaml [hunk 2]
```diff
@@ -714,6 +714,45 @@ plugins:
...
+      - 'IMPACT.esp'
+      - 'IMPACT - ALLDLC.esp'
+      - 'IMPACT-POPMerged.esp'
+  - name: 'JSawyer - Project Nevada.esp'
+    url: [ 'http://www.nexusmods.com/newvegas/mods/60665' ]
+    after:
+      - 'Project Nevada - Cyberware Additions.esp'
+      - 'Project Nevada - All DLC.esp'
+  - name: 'JSawyer - TTW.esp'
+    url: [ 'http://www.nexusmods.com/newvegas/mods/60665' ]
```
Comment (Freso) [inline masterlist.yaml]: We generally don't want entries that *only* contains `url` data.

**Comment by Freso** [comment]
Apart from the two small comments, this PR looks good. Thanks! :)

File: masterlist.yaml [hunk 3]
```diff
@@ -1639,6 +1675,56 @@ plugins:
...
+    after:
+      - 'EVE FNV - ALL DLC.esp'
+      - 'IMPACT.esp'
+      - 'IMPACT - ALLDLC.esp'
+      - 'IMPACT-POPMerged.esp'
+      - 'jsawyer.esp'
+      - 'JSawyer - CE.esp'
+      - 'Project Nevada - Cyberware Additions.esp'
+      - 'Project Nevada - All DLC.esp'
+	  - '357retex.esp'
```
Comment (Freso) [inline masterlist.yaml]: Starts being indented too much here.

Comment (darthbdaman) [inline masterlist.yaml]: It doesn't look like that on my end. That's very odd

Comment (Freso) [inline masterlist.yaml]: Make sure you're not mixing tab indentation and space indentation. (We use space indentation for the `masterlist.yaml` files.)

**Comment by darthbdaman** [comment]
I think I've fixed all the problems and added everything I wanted to now

**Comment by WrinklyNinja** [comment]
This needs a little cleanup (squashing those fixing commits) before I'd be happy to merge.

**Comment by Freso** [comment]
@darthbdaman If you're not up for learning how to squash commits this time around, let me know and I'll do it for you and get the commits merged. :) (I prefer that not everything get squashed together, but thematic commits are still kept individual (e.g., a commit per mod/mod group), so that might be a bit more of an involved exercise. Don't know @WrinklyNinja's preference on that though. ;))

**Comment by WrinklyNinja** [comment]
Ideally, I prefer "one commit changes one aspect" (eg. adding an entry, adding Bash Tags, fixing an issue) so that it's easy to revert if needed, but so long as a commit doesn't do two totally different things it's OK. Sometime's it's tricky to separate changes out into several commits.

Here, it's only squashing those error and indent fixes into the ERP commit. Ideally the Bash Tags commits would get split into lots of smaller commits, but that's not really an issue, and it's hardly something I consistently do myself.

**Comment by darthbdaman** [comment]
If you can squash them that would be great. If you can't I'll try to do it when I get home. We'll see :)

**Comment by WrinklyNinja** [comment]
Merged in 51a863b4c34dd41209f3592e9cc0122037d755a4.

**Comment by WrinklyNinja** [comment]
See, that's the trouble with me doing it, is that the PR gets marked as "closed" instead of "merged".

**Comment by darthbdaman** [comment]
I'll try to do a better job next time :smile:

**Pull request closed without merging**